Document/Example7.ipynb

```diff
@@ -14,6 +14,13 @@
     "# Example-7 (How to plot via seaborn+pandas)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "⚠️ This example is deprecated, you can use `plot` method from `version 3.0`"
```

Comment:
Please change it to:

"⚠️ This example is deprecated. you can use `plot` method from `version 3.0`"